data/autoyast_sle12/create_hdd/create_hdd_sles_regression_s390x.xml.ep

```diff
@@ -0,0 +1,228 @@
+          <usercontrol>no</usercontrol>
+        </interface>
+      </interfaces>
+      <ipv6 config:type="boolean">true</ipv6>
+      <keep_install_network config:type="boolean">true</keep_install_network>
+      <managed config:type="boolean">false</managed>
+      <routing>
+        <ipv4_forward config:type="boolean">false</ipv4_forward>
+        <ipv6_forward config:type="boolean">false</ipv6_forward>
+        <routes config:type="list">
+          <route>
+            <destination>default</destination>
+            <device>eth0</device>
+            <gateway>10.161.159.254</gateway>
+            <netmask>-</netmask>
+          </route>
+        </routes>
+      </routing>
+      <s390-devices config:type="list">
+        <listentry>
+          <chanids>   </chanids>
+          <type/>
+        </listentry>
+        <listentry>
+          <chanids>   </chanids>
+          <type/>
+        </listentry>
+      </s390-devices>
+    </networking>
+    <report>
+        <errors>
+            <log config:type="boolean">true</log>
+            <show config:type="boolean">true</show>
+            <timeout config:type="integer">0</timeout>
+        </errors>
+        <messages>
+            <log config:type="boolean">true</log>
+            <show config:type="boolean">true</show>
+            <timeout config:type="integer">0</timeout>
+        </messages>
+        <warnings>
+            <log config:type="boolean">true</log>
+            <show config:type="boolean">true</show>
+            <timeout config:type="integer">0</timeout>
+        </warnings>
+        <yesno_messages>
+            <log config:type="boolean">true</log>
+            <show config:type="boolean">true</show>
+            <timeout config:type="integer">0</timeout>
+        </yesno_messages>
+    </report>
+    <keyboard>
+        <keyboard_values>
+            <delay/>
+            <discaps config:type="boolean">false</discaps>
+            <numlock>bios</numlock>
+            <rate/>
+        </keyboard_values>
+        <keymap>english-us</keymap>
+    </keyboard>
+    <language>
+        <language>en_US</language>
+        <languages/>
+    </language>
+    <ntp-client>
+        <ntp_policy>auto</ntp_policy>
+    </ntp-client>
+    <software>
+        <products config:type="list">
+            <product><%= uc $get_var->('SLE_PRODUCT') %></product>
+        </products>
+        <packages config:type="list">
+            <!-- Workaround for bsc#1202234: [addon]-release packages are missing after autoyast installation. -->
+            % foreach (values %$addons) {
+            <package><%= lc($_->{name}) %>-release</package>
+            % }
+            <!-- end of workaround -->
+        </packages>
+        <patterns config:type="list">
+           % for my $pattern (@$patterns) {
+           <pattern><%= $pattern %></pattern>
+           % }
+        </patterns>
+    </software>
+    <services-manager>
+        <default_target>graphical</default_target>
```

Comment:
If this is only for graphical, please update the name as create_hdd_gnome_sles_regression_s390x.xml.ep

or you can add the textmode depends on setting of DESKTOP:
% if ($check_var->('DESKTOP', 'gnome')) {
        <default_target>graphical</default_target>
        % }
        % if ($check_var->('DESKTOP', 'textmode')) {
        <default_target>multi-user</default_target>
        % }